src/app/components/Toolbar/Features/Account/ManageableAccountDetails.tsx

```diff
@@ -193,18 +192,14 @@ export const ManageableAccountDetails = ({
                     )}
                   </Text>
                 </Box>
-                {!acknowledge && (
-                  <Button
-                    alignSelf="center"
-                    primary
-                    label={t(
-                      'toolbar.settings.exportPrivateKey.confirm',
-                      'I understand, reveal my private key',
-                    )}
-                    onClick={() => setAcknowledge(true)}
-                  />
-                )}
-                {acknowledge && <PrivateKeyFormatter privateKey={wallet.privateKey!} />}
+                <RevealOverlayButton
```

Comment:
That kinds sucks because priv key is in DOM already without acknowledge.

Reply:
Mocks even show the underlying element blurred, but I think blur might be reversible in screenshots